deploy-board/deploy_board/templates/deploys/confirm_deploy.tmpl

```diff
@@ -24,7 +19,7 @@
                         <div id="confirmDeployModalPrecedingDeployWarningId">
                             Warning
                         </div>
-                        <div id="privateBuildDeployMessageId" class="alert alert-warning" role="alert" hidden=true>
+                        <div id="privateBuildDeployMessageIvvttfgbdkfjtfgbcthfd" class="alert alert-warning" role="alert" hidden=true>
```

Comment:
what is this change for?

Reply:
thanks for finding this -- apologies, looks like my yubikey kicked in

Reply:
this is fixed in the latest diff